### ISSUE TITLE
Error in plugin example_work_with_content_controls_navigation: recursive execution

### ISSUE THREAD

**Issue by flaminestone** [ONLYOFFICE/sdkjs-plugins#38]
Problem with plugin [example_work_with_content_controls_navigation](https://github.com/ONLYOFFICE/sdkjs-plugins/tree/develop/examples/example_work_with_content_controls_navigation)
In [code](https://github.com/ONLYOFFICE/sdkjs-plugins/blob/develop/examples/example_work_with_content_controls_navigation/code.js#L74) was used undocumented method `window.Asc.plugin.event_onTargetPositionChanged`. 

This method will be recursively executed after click to any content control id in plugin meny
![gifout](https://user-images.githubusercontent.com/6047261/46815239-2f227f80-cd83-11e8-8175-407a49d266b1.gif)

**Comment by AlexeyMatveev686** [comment]
Fix commit: https://github.com/ONLYOFFICE/sdkjs-plugins/commit/54c89280ecb543f34a3b77b30b29a274d9f401f2

**Comment by flaminestone** [comment]
Yes, Infinity execution is fixed, but this method is execute after click to any place in page, not only in content control. Next case - try to  mousedown near page and move cursor. Or scroll page - method will be execute too
![gifout](https://user-images.githubusercontent.com/6047261/46879430-c1dc2080-ce4e-11e8-949d-03dec057ce20.gif)

**Comment by Rita-Bubnova** [comment]
@flaminestone, Sorry for the long wait.
Is this issue relevant?

**Comment by flaminestone** [comment]
@Rita-Bubnova Yes, unfortunately, it is still happens. I tested it at now on documentserver-de in docker, version 8.2.

https://github.com/user-attachments/assets/53cd1cc4-7f6f-4866-ba02-33af0ded399b
